cif_core.dic

```diff
@@ -12129,7 +12129,7 @@ save_space_group_wyckoff.letter
 ;
     The Wyckoff letter associated with this position, as given in
     International Tables for Crystallography Volume A. The
-    enumeration value '\a' corresponds to the Greek letter 'alpha'
+    enumeration value 'α' corresponds to the Greek letter 'alpha'
```

Comment:
This sentence does not make a lot of sense with `\a` replaced. However, I think that `\a` enumeration value should in general not be replaced (see my comment in on the `temp_enum.cif` file).

Reply:
I agree with putting it back.

Do the quotes around the \a make it such that it isn't converted to α when it is typeset? Do enumeration values end up being typeset? or used as-is? Or do we need other trickery, like '\\a' to do such a thing? @nautolycus ?

Reply:
IUCr journals practice with regard to enumeration values: usually, cases where they might need to be typeset are in experimental tables or other well-defined contexts. Where appropriate, they may be reformatted on printing to satisfy house style rules (e.g. monoclinic -> Monoclinic, 'Mo K\a' -> 'Mo _K_α' ). There are style rules coded into publcif etc. for doing this.